
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


0it [00:00, ?it/s]1it [00:00,  1.11it/s]2it [00:02,  1.29s/it]3it [00:05,  1.92s/it]4it [00:08,  2.39s/it]5it [00:12,  2.95s/it]6it [00:16,  3.33s/it]7it [00:20,  3.61s/it]8it [00:24,  3.86s/it]8it [00:28,  3.59s/it]
Traceback (most recent call last):
  File "/home/users/delon/aemulusnu_massfunction/prepare_data.py", line 42, in <module>
    assert(len(snapshot_mass)==len(snapshot_Np))
AssertionError
